### PR TITLE
update for publish

### PR DESCRIPTION
**Pull request by aclar48**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
   "homepage": "http://github.com/jira-node/node-jira-client",
   "repository": {
     "type": "git",
-    "url": "http://github.com/jira-node/node-jira-client"
+    "url": "git+ssh://git@github.com/jira-node/node-jira-client.git"
```
Comment (alkazar) [inline package.json]: why this change? arent we trying to get away from ssh?

File: package.json [hunk 2]
```diff
@@ -60,5 +60,12 @@
     "docsRepo": "git@github.com:jira-node/jira-node.github.io.git",
     "docsRoot": "docs/",
     "tmpDocsRepo": "tmp-docs-repo"
+  },
+  "bugs": {
+    "url": "https://github.com/jira-node/node-jira-client/issues"
+  },
+  "directories": {
+    "lib": "lib",
+    "test": "test"
```
Comment (alkazar) [inline package.json]: dont think we need test directory

File: package.json [hunk 3]
```diff
@@ -60,5 +60,12 @@
     "docsRepo": "git@github.com:jira-node/jira-node.github.io.git",
     "docsRoot": "docs/",
     "tmpDocsRepo": "tmp-docs-repo"
+  },
+  "bugs": {
+    "url": "https://github.com/jira-node/node-jira-client/issues"
```
Comment (alkazar) [inline package.json]: although I doubt anyone will actually use this, seems unfair to point people at someone else's project to file bugs.

File: package.json (hunk 1 above)
Comment (aclar48) [reply package.json]: this was created by npm init, which I assume has a good reason for making it that way
